Formula/flux.rb

```diff
@@ -33,6 +33,12 @@ class Flux < Formula
   resource "pkg-config-wrapper" do
     url "https://github.com/influxdata/pkg-config/archive/v0.2.12.tar.gz"
     sha256 "23b2ed6a2f04d42906f5a8c28c8d681d03d47a1c32435b5df008adac5b935f1a"
+
+    livecheck do
+      url "https://github.com/influxdata/pkg-config/tags"
+      regex(%r{href="/influxdata/pkg-config/archive/refs/tags/v(.*).tar.gz"}i)
+      strategy :page_match
+    end
```

Comment:
This always takes the latest though, I don't think we can do that. AFAIK we need to use the one used in the upstream repo.

Reply:
@SMillerDev and @nandahkrishna, sorry for late response to this. I have realised my mistake, and fixed the issue in https://github.com/Homebrew/homebrew-core/pull/112155. I am closing this PR to keep my PR aligned with `Homebrew/core` style as pointed out by @carlocab earlier.